src/command/connector/list.rs

```diff
@@ -1,26 +1,35 @@
-use std::path::PathBuf;
-
+use anyhow::anyhow;
 use clap::Parser;
 use serde::Serialize;
+use std::path::PathBuf;
 
 use crate::composition::supergraph::binary::SupergraphBinary;
 use crate::utils::effect::exec::TokioCommand;
 use crate::{RoverOutput, RoverResult};
 
 #[derive(Debug, Parser, Clone, Serialize)]
 pub struct ListConnector {
-    #[arg(long, value_name = "SCHEMA_PATH")]
-    schema_path: PathBuf,
+    /// The path to the schema file containing the connector.
+    ///
+    /// Optional if there is a `supergraph.yaml` containing only a single subgraph
+    #[arg(long = "schema")]
```

Comment:
maybe keep the `value_name = "SCHEMA_FILE_PATH"` to have better documentation when running the cli

Reply:
I also feel that this command could be moved to a specific submodule like Logging is. Also, we could probably just call the command `Parser`s from our library and do this change there right?